updates/v1.0.1/create_providers_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Winter\Storm\Database\Schema\Blueprint;
+use Winter\Storm\Database\Updates\Migration;
+use Winter\Storm\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('winter_sso_providers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->boolean('is_enabled')->nullable();
+            $table->string('name');
+            $table->string('slug')->index();
```

Comment:
What ties the DB provider record to the socialite driver? Why go with multiple records in a custom table instead of a single settings record containing all the configured providers?